### ISSUE TITLE
Contact result list data inconsistent

### ISSUE THREAD

**Issue by jpbottinelli** [quarentime/quarentime-back-end#25]
When trying to create a new contact, depending on the data sent, the list shows only the information that was sent in the request.
In the first example, the request was based on user_id:
```
{
	"contacts": [
		{
			"user_id": "{{api_test_user_1_user_id}}"
		}
	]
}
```

In the rest they were sent only name and phone number.
```
{
    "result": [
        {
            "date_added": "0001-01-01T00:00:00",
            "status": "healthy",
            "pending": true
        },
        {
            "name": "Edmar 223",
            "phone_number": "+64225056778",
            "date_added": "0001-01-01T00:00:00",
            "status": "healthy",
            "pending": true
        },
        {
            "name": "Test User 1",
            "phone_number": "1234567890",
            "date_added": "0001-01-01T00:00:00",
            "status": "healthy",
            "pending": true
        }
    ],
    "request_id": "12e66bb5-12fe-4e42-aa86-6b0c1222249b"
}
```
There's also the problem of the date_added which is not a real date, even though the format is correct.

One question that I have is what are the mandatory fields for the Contact POST and if the user_id is the key, shouldn't we be showing it?
I saw in the code that there are async calls, so I don't know if there are any validations that we should do during the call, but the GET should show if something went wrong like 'invalid user'or something similar

**Comment by edmarsj** [comment]
Hello @jpbottinelli, I think the problem here is really the validation of required fields. In your example, the first record shouldn't have been allowed on the system, as it is missing "name" and "phone_number" which are the only fields accepted. Another question here can also be: shall we validate the whole request if one contact is invalid or shall we return the success of each individual operation? 

In the meantime, I'm publishing for you a fix failing the whole request if one of the items is broken. 
You should expect to see results similar to this:

![image](https://user-images.githubusercontent.com/20404602/78341271-e3593b00-75f3-11ea-84a8-9ac022628411.png)

Btw, I also fixed the issue with dates, if you still see any with an invalid date, that is very likely to be bad data from the previous versions.

**Comment by jpbottinelli** [comment]
Hey, I think that what you did is ok, we shouldn't allow loading if there's a case that failed.
The problem I think we have is the inconsistency between PersonalInformation endpoint and Contact.
If we're allowing anything to be added as PersonalInformation, we won't be able to validate Contacts, right?
I see the following flow:
POST PersonalInformation with Key fields
POST Contact with same Key Fields but for the contacts
Manage any validation based on those. For example, loading twice the same user shouldn't be allowed, etc)

I'm submitting a couple of issues regarding this so we can discuss

**Comment by vlatkozmejkoski** [comment]
@edmarsj  This should be fixed with latest commit on develop right?

**Comment by ale90bsas** [comment]
@edmarsj @vlatkozmejkoski 

```
{
            "date_added": "0001-01-01T00:00:00",
            "status": "healthy",
            "pending": true
 },
```

We're also defaulting the status to **healthy** and we should have a **Pending** status because we cannot suppose this value.

![image](https://user-images.githubusercontent.com/30640814/78507642-b0c85300-7757-11ea-9321-21757a84bd42.png)

@jpbottinelli FYI

**Comment by edmarsj** [comment]
@jpbottinelli regarding consistency in the /PersonalInformation endpoint I'll fix it with #26 and #27 
I'll also add the Pending health status as default.
